pygin/pygin/src/panel.cpp

```diff
@@ -3,6 +3,27 @@
 #include "panel.hpp"
 #include "far_api.hpp"
 #include "py.integer.hpp"
+#include "python.hpp"
+
+#include <ranges>
+
+namespace {
+
+void WINAPI free_py(void* UserData, const struct FarPanelItemFreeInfo* Info)
```

Comment:
Why do we need `Info` here?

Reply:
We don't need it, this is what FAR declares as a callback. Or did you mean deleting `Info` identifier?

Reply:
Ah, right, it's a callback. Please ignore.